sys/include/net/gnrc.h

```diff
@@ -144,9 +144,10 @@
  * }
  * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
- * @note It is necessary to react with a reply message to the other thread for
- *       @ref GNRC_NETAPI_MSG_TYPE_SET and @ref GNRC_NETAPI_MSG_TYPE_GET
- *       commands
+ * @note When receiving a message of type @ref GNRC_NETAPI_MSG_TYPE_SET or
+ *       @ref GNRC_NETAPI_MSG_TYPE_GET, it is necessary to acknowledge it by
+ *       calling @ref msg_reply() with a message of type
+ *       @ref GNRC_NETAPI_MSG_TYPE_ACK.
```

Comment:
As a newbie I think that's very helpful, but I'm not sure I've correctly understood how it works yet.. By looking at how `gnrc_zep.c:_get()` and `gnrc_zep.c:_event_loop()` work and `gnrc_netapi_get()` is documented, I'd think that the content of the reply message should be either an error code or the size of the content of the message that triggered the reply. Is that the case?